src/Entity/CitizenProject.php

```diff
@@ -215,9 +222,50 @@ public function getPhone(): ?PhoneNumber
         return $this->phone;
     }
 
-    public function getCommittee(): ?Committee
+    public function getCommitteeSupports(): Collection
     {
-        return $this->committee;
+        return $this->committeeSupports;
+    }
+
+    public function getCommitteeSupportsApproved(): Collection
+    {
+        return $this->committeeSupports->filter(function (CitizenProjectCommitteeSupport $c) {
+            return $c->isApprove();
+        });
+    }
+
+    public function setCommitteeSupports(Collection $committeeSupports): void
+    {
+        $this->committeeSupports = $committeeSupports;
+    }
+
+    public function setCommitteesOnSupport(iterable $committees): void
```

Comment:
Is this method used somewhere already? You can either remove it, or use it in the constructor.

Reply:
✔️